WooCommerce/Classes/POS/Presentation/Payments Onboarding/PointOfSaleCardPresentPaymentOnboardingView.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+/// Displays the WooPayments onboarding UI based on the state.
+struct PointOfSaleCardPresentPaymentOnboardingView: View {
+    @ObservedObject var viewModel: PointOfSaleCardPresentPaymentOnboardingViewModel
+
+    var body: some View {
+        VStack(spacing: Constants.verticalSpacing) {
+            HStack {
+                Spacer()
+                Button {
+                    viewModel.cancelOnboarding()
+                } label: {
+                    Image(systemName: "xmark")
+                        .font(.posButtonSymbol)
+                }
+                .accessibilityLabel(Localization.cancelOnboarding)
+                .foregroundColor(Color.posTertiaryText)
+            }
```

Comment:
Consider using the `.posModalCloseButton(action: accessibilityLabel:)` modifier instead – you can just put it on the Onboarding view at the end somewhere and it saves repeating this. See `PointOfSaleCardPresentPaymentScanningForReadersView` for an example

Reply:
Nice, updated in https://github.com/woocommerce/woocommerce-ios/pull/14224/commits/107699e9bae23118124117a6f22bfe99de891d0b.